### ISSUE TITLE
repository.json

### ISSUE THREAD

**Issue by piqus** [TheScriptRunner/repository-channel#1]
The repository channel files should inform about:

- name
- description
- path to repo (to location - http, ftp)
- source: tags or branch
- checksForUpdates?
- operating system, version

The repository channel files should be in separated in files depending on first letter of the script like `a.json`, `b.json`, `c.json` and so on. It will help us handling a future pull requests, plus it is way cleaned up than just one file which i.e. can be too big to open it in browser.

**Comment by AdamJohanssen** [comment]
I don't think this is the best place for this question but I can't figure out where else to ask this. 

Do we want to create another repo at the root of this organization for all the Windows 10 related scripts/packages? It can become the authoritative repo for anyone looking for scripts related to Windows 10 privacy fixes and we can also start working on properly documenting each script/package. It can also be helpful in coming up with guidelines for others to submit their scripts to this channel in the future.

**Comment by piqus** [comment]
@AdamJohanssen  :+1:
